test/unit/gateways/opp_test.rb

```diff
@@ -195,6 +210,14 @@ def successful_response(type, id)
     )
   end
 
+  def successful_store_response(id)
+    OppMockResponse.new(200,
+      JSON.generate({'id' => id, 'result' => {'code' => '000.100.110', 'description' => "Request successfully processed in 'Merchant in Integrator Test Mode'"}, 'card' => {"bin
```

Comment:
Ah, dang it, I see another multiline string here. This should probably be `'bin'`, not `"bin\n"`